src/main/java/ay2021s1_cs2103_w16_3/finesse/model/bookmark/BookmarkIncome.java

```diff
@@ -33,12 +33,12 @@ public boolean equals(Object other) {
             return true;
```

Comment:
The javadoc for this method should be changed from frequent incomes to bookmark incomes (line 27 and 28).